test/test_simulation_results.jl

```diff
@@ -152,11 +162,12 @@ function test_simulation_results(file_path::String, export_path)
             @test size(v) == (12, 6)
         end
 
-        network_duals = read_dual(results_ed, :CopperPlateBalance)
-        @test length(keys(network_duals)) == 48
-        for v in values(network_duals)
-            @test size(v) == (12, 2)
-        end
+        # TODO: duals are not being stored any longer. Is that expected?
```

Comment:
Do you know why this is failing?

Reply:
This should check that the duals in the results have the correct size. If this is redundant we can delete it.

Reply:
```
┌ Error: CopperPlateBalance is not stored
│   sort(existing_names) = Symbol[]
└ @ PowerSimulations ~/sandboxes/PowerSimulations.jl/src/core/simulation_results.jl:157
Test simulation results: Error During Test at /Users/dthom/sandboxes/PowerSimulations.jl/test/test_simulation_results.jl:50
```